### DISCUSSION TITLE
[Kavita] respect right to left reading direction in metadata

### DISCUSSION THREAD

**Discussion by DieselTech** [Kareadita/Kavita#2557] [Ideas]
### Idea Description

Posted by Craig
· Oct 2022
· 13 Upvotes
https://feats.kavitareader.com/posts/231/respect-right-to-left-reading-direction-in-metadata#

Description

Most of my manga have the right-to-left reading direction embedded in their metadata (comicrack) and it kills me that Kavita ignores that information. It is super annoying that every time I start reading I have to manually set the reading direction to R-to-L.

Ideally Kavita would read this data from the metadata (which it is already reading), and also it would allow it to be set manually per series.

### Duration of Using Kavita

42

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by jwillikers** [comment]
Yes, please! https://anansi-project.github.io/docs/comicinfo/documentation#manga
